app/src/Users.vue

```diff
@@ -158,4 +212,22 @@ export default class Users extends Vue {
 .table-page {
   margin: 50px 5% 10px;
 }
-</style>
\ No newline at end of file
+
+.search button {
+  margin-left: 5px;
+}
+
+.search {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+  justify-content: flex-start;
+  width: 100%;
+}
+
+@media screen and (max-width: 200px) {
+  .search {
+    flex-direction: column;
+  }
+}
```

Comment:
is this really needed?

Reply:
It would be needed for smaller screens, as when I resize the output screen, the buttons are not displayed correctly, however, this kind of display width is not so common, so I have removed it in the recent commit.